src/datascience-ui/history-react/interactivePanel.tsx

```diff
@@ -325,13 +325,16 @@ ${buildSettingsCss(this.props.settings)}`}</style>
         containerRef?: React.RefObject<HTMLDivElement>
     ): JSX.Element | null => {
         if (this.props.settings && this.props.editorOptions) {
+            const maxOutputSize = this.props.settings.maxOutputSize;
+            const maxTextSize = maxOutputSize && maxOutputSize < 10000 && maxOutputSize > 0 ? maxOutputSize : undefined;
```

Comment:
This number `10000` is hardcoded in two places, time for a constant.
Also, I'd move the following code into its own function.
```suggestion
            const maxOutputSize = this.props.settings.maxOutputSize;
            const maxTextSize = maxOutputSize && maxOutputSize < 10000 && maxOutputSize > 0 ? maxOutputSize : undefined;
```

Reply:
Done

Reply:
Still not done, number is hardcoded and identical code in two places.

Reply:
By two places do you mean nativeEditor.tsx and interactivePanel.tsx? To reuse it in those 2 places I'd have to create a helpers file just for that one line function, or put it in another file that doesn't make sense. Is that what you want?
For now I used a constant